bin/generate_sdk_documentation.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import docstring_parser
+import inspect
+from pathlib import Path
+import sys
+
+from substra import Client
+from substra.sdk.utils import retry_on_exception
+
+
+MODULE_LIST = [
+    Client,
+    retry_on_exception
+]
+
+KEYWORDS = [
+    "Args",
+    "Returns",
+    "Yields",
+    "Raises",
+    "Example"
+]
+
+
+def generate_function_help(fh, asset):
+    """Write the description of a function"""
+    fh.write(f"# {asset.__name__}\n")
+    signature = str(inspect.signature(asset))
+    fh.write("```python\n")
+    fh.write(f"{asset.__name__}{signature}")
+    fh.write("\n```")
+    fh.write("\n\n")
+    # Write the docstring
+    docstring = inspect.getdoc(asset)
+    docstring = docstring_parser.parse(inspect.getdoc(asset))
+    fh.write(f"{docstring.short_description}\n")
+    if docstring.long_description:
+        fh.write(f"{docstring.long_description}\n")
+    # Write the arguments as a list
+    if len(docstring.params) > 0:
+        fh.write("\nArguments:\n")
+    for param in docstring.params:
+        type_and_optional = ""
+        if param.type_name or param.is_optional is not None:
+            text_optional = 'required'
+            if param.is_optional:
+                text_optional = 'optional'
+            type_and_optional = f"({param.type_name}, {text_optional})"
+        fh.write(f" - {param.arg_name}{type_and_optional}: {param.description}\n")
```

Comment:
Could you make the arg_name and type_name as code (using backquotes) and could you add a space between the arg_name and the type_and_optional? I feel it would improve the readability a bit.